networkx/algorithms/tests/test_similarity.py

```diff
@@ -954,6 +954,16 @@ def test_generate_random_paths_weighted(self):
         assert expected_paths == list(paths)
         assert expected_map == index_map
 
+    def test_one_node_one_loop_and_empty_graph(self):
+        G1 = nx.DiGraph([(0, 0)])
+        G2 = nx.DiGraph()
+        assert nx.graph_edit_distance(G1, G2) == 2
+
+    def test_one_node_two_loops_and_empty_graph(self):
+        G1 = nx.MultiDiGraph([(0, 0), (0, 0)])
+        assert nx.graph_edit_distance(G1, nx.DiGraph()) == 3
+        assert nx.graph_edit_distance(G1, nx.MultiDiGraph()) == 3
+
```

Comment:
```suggestion
        assert nx.graph_edit_distance(G1, nx.MultiDiGraph()) == 3

    def test_two_directed_loops():
        G = nx.DiGraph([(0, 0), (1, 1)])
        assert nx.graph_edit_distance(G, nx.DiGraph()) == 3

```

Reply:
Shouldn't this equal 4? 2 edges + 2 nodes to remove.

Reply:
That seems correct. Sorry about that! 
(Note to self: coding in a suggested change comment is not a good way to avoid bugs!)